models/schedule_week.go

```diff
@@ -0,0 +1,30 @@
+package models
+
+import (
+	"github.com/UniversityRadioYork/myradio-go"
+)
+
+// ScheduleWeekModel is the model for the ScheduleWeek controller.
```

Comment:
This model seems too specific.  The other thing I'm unsure of is whether any of the business logic from above should be in here, but I doubt it.